### PR TITLE
Initial version of kubernetes configs for each service.

### PR DESCRIPTION
**Pull request by calebbrown**
Initial commit of production kubernetes configs for each service needed to run criticality score in production.

Signed-off-by: Caleb Brown <calebbrown@google.com>

### CODE REVIEW

File: infra/config.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  criticality:
+    log-env: gcp
+    log-level: info
+    dataset: ossf_criticality_score_depsdev
+    scoring: enabled
+    scoring-config: config/scorer/pike_depsdev.yml
+    scoring-column-name: default_score
+    output-format: json
+  scorecard:
```
Comment (oliverchang) [inline infra/config.yaml:40]: is this empty scorecard entry needed at all?

File: infra/k8s/worker.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      app.kubernetes.io/name: worker
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: worker
+    spec:
+      containers:
+        - name: worker
+          image: gcr.io/openssf/criticality-score-collect-signals:latest
+          args: ["-config=/etc/criticality_score/config.yaml"]
```
Comment (oliverchang) [inline infra/k8s/worker.yaml:32]: nit: '--' for consistency with the above?

**Review by oliverchang — APPROVED** [review]
LGTM!

File: infra/k8s/worker.yaml (hunk 2 above)
Comment (calebbrown) [reply infra/k8s/worker.yaml]: Done

File: infra/config.yaml (hunk 1 above)
Comment (calebbrown) [reply infra/config.yaml]: Yes, otherwise the controller does not work.

I added a comment.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**